### DISCUSSION TITLE
Differing values with FRiP vs. manual calculation (percentage of reads in peaks)

### DISCUSSION THREAD

**Discussion by matt-sd-watson** [stuart-lab/signac#1049] [Q&A]
I am analyzing joint ATAC + GEX multiome data from 10X with some initial help from the scATAC-seq tutorial: 
https://satijalab.org/signac/articles/pbmc_vignette.html#computing-qc-metrics-1

I am obtaining very different results for the fraction of reads in peaks depending on my approach. 

In one instance, I am using the FRiP approach as shown below: 

fragpath here is to the fragments.tsv.gz file for a particular sample
`total_fragments <- CountFragments(fragpath)`
`seurat_multi[["fragments"]] <- total_fragments[match(colnames(seurat_gex),
                                                     total_fragments$CB),]$frequency_count`
`seurat_multi <- FRiP(
    object = seurat_multi,
    assay = 'ATAC',
    total.fragments = 'fragments',
    col.name = 'pct_reads_in_peaks'
  ) `

Using this approach, the highest value is only 1.5%, which seems very low. 

However, when I manually calculate the values using the following formula, with variables obtained from the 10X metadata output sheet, the values are more consistent with the tutorial: 

`i$rpct_reads_in_peaks <- i$atac_peak_region_fragments / i$atac_fragments * 100`

With this second approach, most of the values are greater than 75%. 

Can someone explain which approach is correct here, and possibly why there is such a discrepancy in these approaches, which Io understood to be equivalent? I can also provide some visualizations to help to explain these differences.

**Comment by timoast** [comment]
Depending on how the matrix was constructed, you might get different results. The `FeatureMatrix()` function in Signac counts fragments in peaks, whereas the matrix reported by cellranger-atac gives cutsite counts per peak. If the matrices used different peak sets then their FRiP values could also be different.
